pages/deployment/github-actions.mdx

```diff
@@ -94,3 +94,30 @@ depending on whether you want the website to be deployed via that protocol.
 The `refresh_token` should contain a Github secret that you set up earlier, 
 and the `site_url` field should contain the subdomain you set up to host your website. 
 
+## Powered By Distributed Press
+
+We'd appreciate if you embedded the following HTML snippit on your DP-published sites to help spread the word.
+What's cool is it will load the site over the same protocol as the one used to access your site.
+
+```html
+<!-- Published with Distributed Press Snippet -->
+<div style="display: inline-block; text-align: center; margin-top: 20px; padding: 8px;">
+  <!-- Powered By Text -->
+  <div>Powered by</div>
+  <!-- Distributed Press Logo -->
+  <a href="//distributed.press" target="_blank" rel="noopener noreferrer">
+    <img
+      alt="Distributed Press"
+      src="//distributed.press/img/logos/logo-distributedpress-grey.png"
+      style="width: 123px; height: auto; margin-top: 10px;"
+    />
+  </a>
+  <!-- Links Below Logo -->
+  <p>
+    <a href="//reader.distributed.press" target="_blank" rel="noopener noreferrer">Follow on ActivityPub</a> |
```

Comment:
Lets remove noreferrer so that we can potentially see where people are coming from for our analytics